### ISSUE TITLE
docker scratch image

### ISSUE THREAD

**Issue by rbucker** [nijtmans/docker-fossil#2]
Is there a way to get fossil to run under a scratch image? Looking at the provided [Dockerfile](https://www.fossil-scm.org/index.html/artifact/f04ebfe52d099660) I see that there is quite a bit going on. Furthermore, what about using a shared volume?

**Comment by nijtmans** [comment]
Yes, it's possible to run fossil on a scratch image: see [http://www.richardbucker.com/2015/02/deploy-fossil-scm-in-docker-container.html]. But the method described here doesn't expose the full fossil possibilities, e.g. https support and Tcl support don't work (yet). This should be solvable, but it takes time to investigate how to do that .....

**Comment by rbucker** [comment]
What's really funny about that link is that I wrote it. Thank you for remembering because I didn't,  Boot2docker clearly does not implement sharing host volumes and tcl is not going to work without including the tcl interpreter which would either have to be embedded into fossil or some other strategy like a shell wrapper or something around fossil. As for https; that should work as-is so long as the SSL code is in fossil, with the certs copied to the container or a shared volume, I'm not certain it's rational but putting the certs into the container will make them vulnerable.
